packages/builders/scripts/testing/recorded-retired-instances.js

```diff
@@ -0,0 +1,67 @@
+import { makeTracer } from '@agoric/internal';
+import { E } from '@endo/far';
+
+const trace = makeTracer('RecordedRetired', true);
+
+/**
+ * @param {BootstrapPowers &
+ *   PromiseSpaceOf<{ retiredContractInstances: MapStore<string, Instance>;
+ *   }>
+ * } powers
+ */
+export const testRecordedRetiredInstances = async ({
+  consume: {
+    contractKits,
+    retiredContractInstances: retiredContractInstancesP,
+  },
+}) => {
+  trace('Start');
+  const retiredContractInstances = await retiredContractInstancesP;
```

Comment:
we do this a lot. I wonder if it would be better not to destructure `consume` in the params so we could do,
```
  const { contractKits, retiredContractInstances } = deeplyFulfilled(consume);
```
totally optional

Reply:
deeplyFulfilled(consume) sounds bad. At best, it would try to grab everything in the space. But I suspect the properties are not even enumerable.

Reply:
oh right. we'd need something like `await space(consume, ['retiredContractInstances')`